examples/web/src/profiles/ProfilesPage.tsx

```diff
@@ -3,81 +3,91 @@ import { LinkCard } from '../components/LinkCard';
+    description: 'Fetch a single profile.',
     path: '/profiles/useProfile',
   },
   {
     label: 'useLazyProfile',
-    description: `Lazy fetch a single profile.`,
+    description: 'Lazy fetch a single profile.',
     path: '/profiles/useLazyProfile',
   },
   {
     label: 'useProfiles',
-    description: `Fetch a list of profiles.`,
+    description: 'Fetch a list of profiles.',
     path: '/profiles/useProfiles',
   },
   {
     label: 'useProfileFollowers',
-    description: `Fetch a list of profile followers.`,
+    description: 'Fetch a list of profile followers.',
     path: '/profiles/useProfileFollowers',
   },
   {
     label: 'useProfileFollowing',
-    description: `Fetch a list of profile following.`,
+    description: 'Fetch a list of profile following.',
     path: '/profiles/useProfileFollowing',
   },
   {
     label: 'useMutualFollowers',
-    description: `Fetch a list of mutual followers between profiles.`,
+    description: 'Fetch a list of mutual followers between profiles.',
     path: '/profiles/useMutualFollowers',
   },
   {
     label: 'useFollow & useUnfollow',
-    description: `Follow and unfollow a profile.`,
+    description: 'Follow and unfollow a profile.',
     path: '/profiles/useFollow',
   },
   {
     label: 'useRecommendedProfiles',
-    description: `Fetch a list of recommended profiles.`,
+    description: 'Fetch a list of recommended profiles.',
     path: '/profiles/useRecommendedProfiles',
   },
   {
     label: 'useWhoActedOnPublication',
-    description: `Fetch a list of profiles who acted on a publication.`,
+    description: 'Fetch a list of profiles who acted on a publication.',
     path: '/profiles/useWhoActedOnPublication',
   },
   {
     label: 'useProfileActionHistory',
-    description: `Fetch profile action history.`,
+    description: 'Fetch profile action history.',
     path: '/profiles/useProfileActionHistory',
   },
   {
     label: 'useProfileManagers',
-    description: `Fetch a list of profile managers.`,
+    description: 'Fetch a list of profile managers.',
     path: '/profiles/useProfileManagers',
   },
   {
     label: 'useUpdateProfileManagers',
-    description: `Update the Profile Manager configuration for the logged-in Profile.`,
+    description: 'Update the Profile Manager configuration for the logged-in Profile.',
     path: '/profiles/useUpdateProfileManagers',
   },
   {
     label: 'useSetProfileMetadata',
-    description: `Set the metadata for the authenticated Profile.`,
+    description: 'Set the metadata for the authenticated Profile.',
     path: '/profiles/useSetProfileMetadata',
   },
   {
     label: 'useUpdateFollowPolicy',
-    description: `Update the follow policy for the logged-in Profile.`,
+    description: 'Update the follow policy for the logged-in Profile.',
     path: '/profiles/useUpdateFollowPolicy',
   },
+  {
+    label: 'useUnblockProfiles',
+    description: `Unblock profiles.`,
+    path: '/profiles/useUnblockProfiles',
```

Comment:
do we need this separately from the 'useBlockProfiles & useUnblockProfiles' item below in this list?